src/geometryRTheta/advection/README.md

```diff
@@ -169,20 +170,10 @@ A = - E \wedge e_z = -\nabla \phi \wedge e_z.
 
 In [the documentation for the advection field](./../advection_field/README.md), we show that 
 ```math
-\nabla_{xy} \phi = J D_{G}^{-1}\nabla_{r\theta} \phi,
+\nabla_{xy} \phi = J \nabla_{r\theta} \phi,
 ```
 
-with *J* the Jacobian matrix and the following diagonal matrix 
-```math
-D_{G} 
-= 
-\begin{bmatrix}
-    \sqrt{g_{11}} & 0 \\
-    0 & \sqrt{g_{22}} \\
-\end{bmatrix}
-```
-
-with the metric tensor $`G = J^TJ = [g_{ij}]_{ij}`$. 
+with *J* the Jacobian matrix (and the metric tensor $`G = J^TJ = [g_{ij}]_{ij}`$). 
```

Comment:
I don't understand this notation `[g_{ij}]_{ij}`. Why does `ij` appear twice?

Reply:
This is also strange in the main branch though so I am not going to delay the PR for this